RustCrypto/Cargo.toml

```diff
@@ -0,0 +1,41 @@
+[package]
+authors = ["Brian Smith <brian@briansmith.org>"]
+name = "crypto_bench_rustcrypto"
+version = "0.1.0"
+
+[lib]
+name = "crypto_bench_rustcrypto"
+path = "RustCrypto.rs"
+
+[dependencies.crypto_bench]
+path = "../crypto_bench"
+
+[dependencies]
+#sha2 = { git = "https://github.com/RustCrypto/hashes" }
+sha2 = { git = "https://github.com/RustCrypto/hashes", features = ["asm"] }
+
```

Comment:
Everything below this point should be removed since this was all moved to the top-level Cargo.toml.

Reply:
done